app/scenes/Settings/ApiKeys.tsx

```diff
@@ -23,6 +24,23 @@ function ApiKeys() {
   const can = usePolicy(team);
   const context = useActionContext();
 
+  const [copiedKeyId, setCopiedKeyId] = React.useState<string | null>();
+  const copyTimeoutIdRef = React.useRef<ReturnType<typeof setTimeout>>();
+
+  const handleCopy = React.useCallback(
+    (keyId: string) => {
+      if (copyTimeoutIdRef.current) {
+        clearTimeout(copyTimeoutIdRef.current);
+      }
+      setCopiedKeyId(keyId);
+      copyTimeoutIdRef.current = setTimeout(() => {
+        setCopiedKeyId(null);
+      }, 3000);
+      toast.message(t("API token copied to clipboard"));
```

Comment:
```suggestion
      toast.message(t("API key copied to clipboard"));
```

Reply:
This breaks the existing translations.
Do we want to do this?

Reply:
Yes it's ok, thanks